Source/WTF/wtf/MemoryPressureHandler.cpp

```diff
@@ -95,6 +95,12 @@ static const char* toString(MemoryUsagePolicy policy)
 }
 #endif
 
+static size_t thresholdForMemoryKillOfActiveProcess(unsigned tabCount)
+{
+    size_t baseThreshold = ramSize() > 16 * GB ? 15 * GB : 7 * GB;
```

Comment:
Change log says:
> This patch adds a foreground memory limit of 8GB or 16GB depending on RAM size

but seems like the code does something a bit different? Even if you assume there is a single tab in the process,
seems like 16GB of RAM would give you a limit of 8GB, no? Since you are using > and not >= in your check.

Reply:
Oh, I misread. Your change log only says it's based on RAM size but no actual values.

Reply:
Right, the `>` rather than `>=` here is intentional to match the magic values that other browsers chose based on RAM size.